01-data-structures/03-linked-lists/linked-lists-answers.txt

```diff
@@ -0,0 +1 @@
+1. Spatial Locality is the process by which the computer, (when searching for a piece of data not present in the cache,) rather than retrieving a specific piece of data from the RAM, retrieves a range of data (memory locations.) The reason this benefits performance is because while it would seem inefficient to pull a larger amount of data and sort through it, the bottleneck in speed is actually retrieving data from RAM, and by grabbing larger partitions, it reduces the number of times the computer needs to take this action. 
```

Comment:
It is more that spatial locality is the reason the computers are designed to do that. Spatial locality actually refers to the observation that often times, due to the design of arrays, objects, and procedure stack frames, memory locations close to one another will be accessed within a small time frame. Using that observation, CPU-side caches have been added to improve performance. If spatial locality was not true, then there would be no benefit to the actual process of caching.